budget-squirrel/src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const proxy = require('http-proxy-middleware');
+
+module.exports = function(app) {
+  app.use(
+    '/api',
+    proxy({
+        // This needs to be set to whatever our endpoint is.
+      target: 'http://localhost:5000',
```

Comment:
Let's talk about this in person. I want to make sure I know what it does. Axios looks really nice though. Good find.